bazel.sh

```diff
@@ -30,6 +30,8 @@
 #   -v|--verbose:                     Generates verbose output on local runs.
 #                                     Does not affect kokoro runs.
 #
+# Any other unrecognized arguments will be passed along to the bazel invocation.
```

Comment:
I think you should keep 'other' or 'unrecognized' but putting them both together makes it sound like the ones above are also unrecognized.

Reply:
Hah good call, fixed!